lib/command/definitions.js

```diff
@@ -2,160 +2,61 @@ const getConfig = require('./utils').getConfig;
-  fails(): ScenarioConfig;
-  retry(times: number): ScenarioConfig
-  timeout(timeout: number): ScenarioConfig
-  inject(inject: object): ScenarioConfig
-  config(config: object): ScenarioConfig
-  config(helperName: string, config: object): ScenarioConfig
-  injectDependencies(dependencies: { [key: string]: any }): ScenarioConfig
-}
+type translation = ${JSON.stringify(translations.vocabulary, null, 2)};
+type WithTranslation<T> = T & Translate<T, translation["actions"]>;
+type methods = ${helperNames.length > 0 ? helperNames.join(' & ') : 'undefined'};
 
-interface ILocator {
-  id?: string;
-  xpath?: string;
-  css?: string;
-  name?: string;
-  frame?: string;
-  android?: string;
-  ios?: string;
-}
-
-type LocatorOrString = string | ILocator | Locator;
-
-declare class Helper {
-  /** Abstract method to provide required config options */
-  static _config(): any;
-  /** Abstract method to validate config */
-  _validateConfig<T>(config: T): T;
-  /** Sets config for current test */
-  _setConfig(opts: any): void;
-  /** Hook executed before all tests */
-  _init(): void
-  /** Hook executed before each test. */
-  _before(): void
-  /** Hook executed after each test */
-  _after(): void
-  /**
-   * Hook provides a test details
-   * Executed in the very beginning of a test
-   */
-  _test(test: () => void): void
-  /** Hook executed after each passed test */
-  _passed(test: () => void): void
-  /** Hook executed after each failed test */
-  _failed(test: () => void): void
-  /** Hook executed before each step */
-  _beforeStep(step: () => void): void
-  /** Hook executed after each step */
-  _afterStep(step: () => void): void
-  /** Hook executed before each suite */
-  _beforeSuite(suite: () => void): void
-  /** Hook executed after each suite */
-  _afterSuite(suite: () => void): void
-  /** Hook executed after all tests are executed */
-  _finishTest(suite: () => void): void
-  /**Access another configured helper: this.helpers['AnotherHelper'] */
-  readonly helpers: any
-  /** Print debug message to console (outputs only in debug mode) */
-  debug(msg: string): void
-
-  debugSection(section: string, msg: string): void
-}
-
-declare class Locator {
-  constructor(locator: LocatorOrString, defaultType?: string);
-
-  or(locator: LocatorOrString): Locator;
-  find(locator: LocatorOrString): Locator;
-  withChild(locator: LocatorOrString): Locator;
-  withDescendant(locator: LocatorOrString): Locator;
-  at(position: number): Locator;
-  first(): Locator;
-  last(): Locator;
-  inside(locator: LocatorOrString): Locator;
-  before(locator: LocatorOrString): Locator;
-  after(locator: LocatorOrString): Locator;
-  withText(text: string): Locator;
-  withAttr(attrs: object): Locator;
-  as(output: string): Locator;
+declare namespace CodeceptJS {
+  export type I = ${customFilePaths.I ? 'ReturnType<typeof I>' : 'WithTranslation<methods>'};
 }
 
+type ICodeceptCallback = (i: CodeceptJS.I${callbackParams}) => void;
```

Comment:
This callback has a hard order of arguments. And CodeceptJS allows you to transfer here PageObjects in an arbitrary order. And when I don't want to use I but only PageObject, autocomplete suggests methods for I